spec/services/cocina_generator/dro_generator_spec.rb

```diff
@@ -395,6 +403,27 @@
             },
             structural: {
               contains: [
+                {
+                  label: 'MyString',
+                  structural: { contains: [
+                    {
+                      access: { view: 'world', download: 'world' },
+                      administrative: { publish: true, sdrPreserve: true, shelve: true },
+                      filename: 'favicon.ico',
+                      hasMessageDigests: [
+                        { digest: 'aa9cf3e587a33b77b6d6ee0f8d36de39', type: 'md5' },
+                        { digest: 'e30e483ba4222573e8952ca066af8d2a1bd0bc53', type: 'sha1' }
+                      ],
+                      hasMimeType: 'image/png',
+                      label: 'MyString',
+                      size: 4_045,
+                      type: Cocina::Models::ObjectType.file,
+                      version: 1
+                    }
+                  ] },
+                  type: Cocina::Models::FileSetType.file,
+                  version: 1
+                },
```

Comment:
This test fails when the `sort_by` is removed in the structural generator. This shows that the order of filesets is based not on the order of uploads but on the filename.